### DISCUSSION TITLE
Where is the deepest* mailbox in the world?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#557] [Galaxy Trivia] [answered]
**Bonus:** How did it get there?

**Comment by SebastienXia** [comment]
The "deepest mailbox in the world" in Susami Bay, Japan was placed there intentionally in 1999 by the local government as a way to promote tourism and provide a unique experience for scuba divers.

**Comment by thuongtruong1009** [comment, marked as answer]
The deepest mailbox in the world is located in Susami Bay, a few miles off the coast of the city of Susami in Wakayama Prefecture, Japan. The mailbox is situated underwater, about 10 meters (33 feet) below the surface. It was installed by the local post office in 1999 as a tourist attraction, and visitors can dive or snorkel to reach the mailbox and deposit their mail. The mailbox is made of stainless steel and is designed to be water-resistant to protect the letters inside. It has since become a popular spot for divers and tourists from around the world to send postcards to their loved ones, making it a unique and unusual destination.
